app/config/collections.php

```diff
@@ -2076,7 +2076,7 @@
                 '$id' => ID::custom('name'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 2048,
+                'size' => 128,
```

Comment:
We might need to allow for more characters in the name. 128 may be limiting

Reply:
I consult this with Torsten, we can change it up to 768 without any problems, If we need more I will have to restrict the index length